### PR TITLE
New test file plus instructions to make more

### PR DESCRIPTION
**Pull request by rsheeter**
As discussed in f2f this morning. ttx_diff on this doesn't match and is tractably small. JMM.

### CODE REVIEW

File: resources/testdata/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+* `resources/testdata/glyphs3/Oswald-AE-comb.glyphs`
+   * Oswald with most of the glyphs deleted
+   * After deletion go to Info > Features
+      * Update language systems
+      * Delete features
```
Comment (cmyr) [inline resources/testdata/README.md:5]: this is all already done I imagine? Are these instructions intended as general instructions for how to make a test font like this?

Comment (rsheeter) [reply resources/testdata/README.md:5]: For that or if you want to build a new version of this file with more glyphs

Comment (rsheeter) [reply resources/testdata/README.md:5]: In general I've never regretted capturing how a given test file was produced and have definitely regretted *not* doing so :)

Comment (cmyr) [reply resources/testdata/README.md:5]: oh definitely, I was just confused by the wording. :)

**Pull request merged**